samples/subscriptions.js

```diff
@@ -81,11 +81,11 @@ async function createSubscription(topicName, subscriptionName) {
   // [END pubsub_create_pull_subscription]
 }
 
-async function createFlowControlledSubscription(
+async function subscribeWithFlowControlSettings(
   topicName,
```

Comment:
```suggestion
```

Reply:
`topicName` isn't relevant since subscriptions are already mapped to a topic. We can remove these from the sample.